src/commands/registerCommands.ts

```diff
@@ -62,32 +56,27 @@ export function registerCommands(): void {
     registerCommand('appService.appSettings.ToggleSlotSetting', toggleSlotSetting);
     registerCommand('appService.appSettings.Upload', uploadAppSettings);
     registerCommand('appService.Browse', browseWebsite);
-    registerCommand('appService.CloneTrialApp', cloneTrialApp);
     registerCommand('appService.ConfigureDeploymentSource', editScmType);
     registerCommand('appService.connectToGitHub', connectToGitHub);
     registerCommand('appService.CreateSlot', createSlot);
-    registerCommand('appService.CreateTrialApp', createTrialApp);
     registerCommand('appService.CreateWebApp', createWebApp);
     registerCommand('appService.CreateWebAppAdvanced', createWebAppAdvanced);
     registerCommand('appService.Delete', deleteWebApp);
     registerCommand('appService.DeploymentScript', generateDeploymentScript);
     registerCommand('appService.DisconnectRepo', disconnectRepo);
```

Comment:
> You could get rid of the errors.ts file as well if you changed disconnectRepo since it was only using the OperationNotSupportedError in case the parent was a trial app.

I didn't think of this, but now I feel like I'd rather leave it as is. I don't want to get rid of `node.parent instanceof SiteTreeItem` in `disconnectRepo` because I'd have to cast `parent` to `SiteTreeItem` and I don't like casting as much. Also, `errors.ts` is a common enough pattern across all our repos that I don't mind keeping it even if it's not super useful.